gulpfile.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
```

Comment:
@loganlsr Gulpfile looks good - feel free to add additional tasks as needed - this file is meant to automate tasks that you need for your own workflow